src/MongoDB.AspNetCore.OData/MongoDB.AspNetCore.OData.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
We do need Web SDK, as there are usages of HttpRequest and some another objects that is belong to Web SDK. I've suppressed usage of launch settings though. Please let me know if this solves VS problems.

Reply:
We can get rid of the Web SDK, but in this case we have to add bunch of conditional references for each asp.net version to the ref package. This would not make our life easier =)